common/app/com/commercetools/sunrise/myaccount/CustomerSessionHandler.java

```diff
@@ -0,0 +1,65 @@
+package com.commercetools.sunrise.myaccount;
+
+import com.commercetools.sunrise.common.sessions.SessionHandlerBase;
+import com.google.inject.Injector;
+import io.sphere.sdk.customers.Customer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import play.mvc.Http;
+
+import javax.inject.Inject;
+import java.util.Optional;
+
+import static java.util.Arrays.asList;
+
+public class CustomerSessionHandler extends SessionHandlerBase<Customer> {
+
+    private static final Logger logger = LoggerFactory.getLogger(CustomerSessionHandler.class);
+    private static final String CUSTOMER_ID_SESSION_KEY = "sunrise-customer-id";
```

Comment:
But currently you could simply do it programatically (extend the class and overwrite the session key method) and bind your session class instead, so it's not impossible to do. It's just simply not supported via configuration yet. So instead of programatically I can switch it to configuration, but it was possible all the time.